src/pages/AddPage/AddPage.jsx

```diff
@@ -1,7 +1,46 @@
 import React from "react";
+import "./AddPage.scss";
+import { useState } from "react";
 
 const AddPage = () => {
-  return <h2>add</h2>;
+  const url = "http://localhost:3030";
```

Comment:
Create a file and name it `config.js`. Export this variable. and Import it here. Also `url` is a very generic variable name. Rename it to something like: `apiBaseUrl`.

Reply:
Done.